source/index.html

```diff
@@ -1,12 +1,214 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="header">
+      <h1 class="visually-hidden">Пинк - мобильное приложение для раскрашивания серых будней в розовый цвет.</h1>
+      <a class="header__logo logo" href="#">
+        <img src="https://via.placeholder.com/74x23" alt="Логотип Пинк">
+      </a>
+      <nav class="header-nav">
+        <button class="header-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню.</span>
+        </button>
+        <div class="header-nav__wrapper site-nav">
+          <ul class="site-nav__list nav-list">
+            <li class="nav-list__item nav-list__item--current">
+              <a class="nav-list__link" href="#">Главная</a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="#">Фотографии</a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="#">Конкурс</a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="https://htmlacademy.ru">Html Academy</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <section class="download-app">
+        <h2 class="visually-hidden">Скачать приложение на своё устройство.</h2>
+        <p class="download-app__slogan">Взгляните на жизнь иначе!</p>
+        <div class="download-app__wrapper">
+          <a class="download-app__link button-primary download-button" href="#">Скачать приложение</a>
+          <ul class="download-app__list platforms-list">
+            <li class="platforms-list__item">
+              <img src="../img/platforms/apple-logo.svg" width="47" height="56" alt="Логотип Apple">
+            </li>
+            <li class="platforms-list__item">
+              <img src="../img/platforms/android-logo.svg" width="43" height="47" alt="Логотип Android">
+            </li>
+            <li class="platforms-list__item">
+              <img src="../img/platforms/microsoft-logo.svg" width="44" height="41" alt="Логотип Microsoft">
+            </li>
+          </ul>
+          <p class="download-app__description">
+            Доступно для iPhone, iPad, Android, Windows Phone, OS X, Windows 8
+          </p>
+        </div>
+      </section>
+      <section class="product-features">
+        <h2 class="visually-hidden">Основные особенности товара.</h2>
+        <div class="product-features__subtitle subtitle-wrapper">
+          <p class="subtitle">Устали от серости мегаполиса?
+
+            Нам есть, что вам
+            предложить!</p>
```

Comment:
Какое-то странное форматирование. Можно так:
```suggestion
          <p class="subtitle">
            Устали от серости мегаполиса?
            Нам есть, что вам предложить!
          </p>
```

Ну или так:
```suggestion
          <p class="subtitle">Устали от серости мегаполиса? Нам есть, что вам предложить!</p>
```

А вообще, судя по форматированию текста, параграфов тут 2